v22.2/row-level-ttl.md

```diff
@@ -351,50 +418,22 @@ SELECT * FROM events WHERE crdb_internal_expiration > now();
 (3 rows)
 ~~~
 
-### Remove Row-Level TTL from a table
-
-To drop the TTL on an existing table, reset the [`ttl` storage parameter](#param-ttl).
+To fetch only those rows from a table with [per-row TTL](#create-a-table-with-per-row-ttl) that have not yet expired their TTL, use the `expired_at` column you created earlier:
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-ALTER TABLE events RESET (ttl);
+SELECT * FROM ttl_test_per_row WHERE expired_at > now();
 ~~~
 
-### Set the Row-Level TTL for an individual row
-
-Setting the TTL for an individual row can be useful for compliance reasons, or when troubleshooting your TTL settings.
-
-To set (or change) the TTL for an individual row, issue a query like the one shown below. It updates the expiration of one of the rows you entered when you [created the table](#create-a-table-with-row-level-ttl) to use an [interval](interval.html) of `'1 year'`.
-
-{% include_cached copy-clipboard.html %}
-~~~ sql
-UPDATE events SET crdb_internal_expiration = current_timestamp():::TIMESTAMPTZ + '1 year'::INTERVAL where id = 'a9404386-c4da-415f-b0b0-0dfad0f13c80';
-~~~
-
-~~~
-UPDATE 1
-~~~
+### Remove Row-Level TTL from a table
 
-You can view the new expiration by entering the following query:
+To drop the TTL on an existing table, reset the [`ttl` storage parameter](#param-ttl).
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-SELECT *, crdb_internal_expiration from events WHERE id = ' a9404386-c4da-415f-b0b0-0dfad0f13c80';
-~~~
-
-~~~
-                   id                  | description |        inserted_at         |   crdb_internal_expiration
----------------------------------------+-------------+----------------------------+--------------------------------
-  a9404386-c4da-415f-b0b0-0dfad0f13c80 | a thing     | 2022-04-19 19:53:00.037216 | 2023-04-19 19:57:27.592898+00
-(1 row)
+ALTER TABLE events RESET (ttl);
```

Comment:
You can also do `ALTER TABLE events RESET (ttl_expire_after);` or `ALTER TABLE events RESET (ttl_expiration_expression);` if both are set and you want to remove one or the other.

Reply:
Added this info - PTAL